### ISSUE TITLE
Disabling locked toolkit.telemetry.enabled

### ISSUE THREAD

**Issue by KOLANICH** [The-OP/Fox#156]
`toolkit.telemetry.enabled` is a pref controlling telemetry verbosity, it is locked, and the lock is hardcoded into C++ code to prevent it from being disabled easily https://dxr.mozilla.org/mozilla-central/source/modules/libpref/Preferences.cpp#3213

When [they have been asked about that but they have rejected the proposal to unlock it](https://bugzilla.mozilla.org/show_bug.cgi?id=1422689).

Fortunately, in https://github.com/ghacksuserjs/ghacks-user.js/issues/271#issuecomment-398524920 the user @earthlng have advised to use autoconfig to disable this pref.

Autoconfig is described in https://support.mozilla.org/en-US/kb/customizing-firefox-using-autoconfig and some better docs is in https://github.com/Theemim/GeckoAutoconfigIntro .

So here is the manual:
1. create a file `<firefox install dir>/defaults/pref/autoconfig.js` with the text
```javascript
pref("general.config.obscure_value", 0);
pref("general.config.filename", "firefox.cfg.js");
```

Any other prefs won't work in this file.

2. create the file `<firefox install dir>/firefox.cfg.js` with the text
```javascript
//
lockPref("toolkit.telemetry.enabled", false);
```

and any other JavaScript code you want to execute.

In Firefox 62.0b1 this lock changes the locked value.

3. PROTECT THE FILES FROM NON-ADMIN ACCESS!

**Comment by Atavic** [comment]
Valuable info :+1: ...while that bugzilla really made me sad.
:-1: for Mozilla.

**Comment by Atavic** [comment]
The more I look at bugzilla, the more its community appears as a place where their products are destroyed under will: https://bugzilla.mozilla.org/show_bug.cgi?id=222973

**Comment by leedoyle** [comment]
Thank you, @KOLANICH 
I hope we support updating the repository at least in the form of creating new issues.

**Comment by KOLANICH** [comment]
I wonder if this repo makes any sense. Should we migrate to ghacksuserjs/ghacks-user.js add our content to their and just translate the result?

**Comment by Atavic** [comment]
@KOLANICH maybe [pyllyukko](https://github.com/pyllyukko/user.js)?

**Comment by KOLANICH** [comment]
@Atavic , moved the cursor to star button only to see that this repo is already starred ;) But ghacksuserjs/ghacks-user.js looks more active. I mean I have subscribed to it and I see the messages from it every day.

**Comment by Atavic** [comment]
Yes, that repo is very active. (That's why I suggested the less active one)

**Comment by Atavic** [comment]
BTW This repo makes sense for russian users. Most of the other ones are in english.

**Comment by Lucatir** [comment]
Hello. My privacy invading friend Google brought me here. Can still confirm that this works on version 70.

For men of the penguin,  at least in Arch based linux distributions, the install folder is in /usr/lib/
You're welcome, fam.
